app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by(id: params[:id]) 
+    render html:"User not found" if @user.nil?
+    @user
```

Comment:
thay line 8-9 thành
`return if @user`
`render html:"User not found"` 

`"User not found"` -> i18n nhé